src/app/dataset-view/additional-components/dataset-settings-component/tabs/scheduling/dataset-settings-scheduling-tab.component.ts

```diff
@@ -0,0 +1,227 @@
+import { MaybeNull } from "../../../../../common/app.types";
+import { ChangeDetectionStrategy, Component, Input, OnInit } from "@angular/core";
+import { AbstractControl, FormControl, FormGroup, Validators } from "@angular/forms";
+import { BaseComponent } from "../../../../../common/base.component";
+import { PollingGroupEnum, ThrottlingGroupEnum } from "../../dataset-settings.model";
+import {
+    DatasetBasicsFragment,
+    DatasetFlowType,
+    DatasetKind,
+    DatasetPermissionsFragment,
+    ScheduleInput,
+    TimeUnit,
+} from "src/app/api/kamu.graphql.interface";
+import { DatasetSchedulingService } from "../../services/dataset-scheduling.service";
+import { cronExpressionValidator } from "src/app/common/data.helpers";
+import { cronExpressionNextTime, logError } from "src/app/common/app.helpers";
+
+@Component({
+    selector: "app-dataset-settings-scheduling-tab",
+    templateUrl: "./dataset-settings-scheduling-tab.component.html",
+    styleUrls: ["./dataset-settings-scheduling-tab.component.scss"],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class DatasetSettingsSchedulingTabComponent extends BaseComponent implements OnInit {
+    @Input() public datasetBasics: DatasetBasicsFragment;
+    @Input() public datasetPermissions: DatasetPermissionsFragment;
+    public readonly pollingGroupEnum: typeof PollingGroupEnum = PollingGroupEnum;
+    public readonly throttlingGroupEnum: typeof ThrottlingGroupEnum = ThrottlingGroupEnum;
+    public readonly timeUnit: typeof TimeUnit = TimeUnit;
+    private scheduleOptions: ScheduleInput;
+
+    public pollingForm = new FormGroup({
+        updatesState: new FormControl(false),
+        pollingGroup: new FormGroup({
+            __typename: new FormControl(PollingGroupEnum.TIME_DELTA, [Validators.required]),
+            every: new FormControl({ value: null, disabled: true }, [Validators.required]),
+            unit: new FormControl({ value: "", disabled: true }, [Validators.required]),
+            cronExpression: new FormControl({ value: "", disabled: true }, [
+                Validators.required,
+                cronExpressionValidator(),
+            ]),
+        }),
+    });
+
+    public throttlingForm = new FormGroup({
+        every: new FormControl({ value: 1, disabled: true }, [Validators.required]),
+        unit: new FormControl({ value: TimeUnit.Minutes, disabled: true }, [Validators.required]),
```

Comment:
Reusing "every" and "unit" for throttling might be a bad idea.
Unlike root datasets time delta, in derived datasets these fields are optional.

Reply:
Fixed